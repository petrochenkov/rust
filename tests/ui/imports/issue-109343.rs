#![crate_type = "lib"]

pub mod f {}
pub use unresolved::f;
//~^ ERROR unresolved import `unresolved`

/// [g]
pub use f as g;
//~^ ERROR cannot determine resolution for the import
//~| ERROR cannot determine resolution for the import

fn main() {}
