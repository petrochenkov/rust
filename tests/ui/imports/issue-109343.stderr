error: cannot determine resolution for the import
  --> $DIR/issue-109343.rs:8:9
   |
LL | pub use f as g;
   |         ^^^^^^

error: cannot determine resolution for the import
  --> $DIR/issue-109343.rs:8:9
   |
LL | pub use f as g;
   |         ^^^^^^
   |
   = note: duplicate diagnostic emitted due to `-Z deduplicate-diagnostics=no`

error[E0432]: unresolved import `unresolved`
  --> $DIR/issue-109343.rs:4:9
   |
LL | pub use unresolved::f;
   |         ^^^^^^^^^^ use of unresolved module or unlinked crate `unresolved`
   |
help: you might be missing a crate named `unresolved`, add it to your project and import it in your code
   |
LL + extern crate unresolved;
   |

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0432`.
